pkg/operator/ceph/csi/template/multus-host-setup.yaml

```diff
@@ -0,0 +1,47 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-multus
+  namespace: {{ .Namespace }}
+spec:
+  selector:
+    matchLabels:
+      name: rook-ceph-multus
+  updateStrategy:
+    type: OnDelete
```

Comment:
If we ever have a critical update to this new multus daemonset, how will we update it? It will be disruptive to all the application pods, right?

Reply:
Updating this daemonset will be disruptive to application pods. The best way would be to delete the daemonset pod and reboot the node. Otherwise the multus interface in the host namespace will need to be deleted; after the daemonset set pods are deleted and the updated pods are deployed, a new interface with a new IP will be added to it. If the new IP isn't the same the csi pods will face the same issue that this PR aims to fix.